app/mixins/google-pageview.js

```diff
@@ -1,13 +1,14 @@
 import Ember from 'ember';
 
 export default Ember.Mixin.create({
+
+    metrics: Ember.inject.service(),
+
     notifyGoogleAnalytics: Ember.on('didTransition', function() {
-        if (!window.ga) {
-            return;
-        }
-        return window.ga('send', 'pageview', {
-            page: this.get('url'),
-            title: this.get('url')
+        Ember.run.scheduleOnce('afterRender', this, () => {
+            const page = this.get('url');
+            const title = this.getWithDefault('currentRouteName', 'unknown');
```

Comment:
`title` had been set to `this.get('url')` too until now. maybe we should keep it like that for now and change that in a separate PR?